applications/app/views/fragments/crosswords/crosswordMetaHeader.scala.html

```diff
@@ -37,7 +37,11 @@ <h1 itemprop="headline" class="content__headline js-score">@Html(crosswordPage.c
                     <div class="meta__save-for-later js-save-for-later" data-position="top"></div>
                 }
                 <div class="meta__numbers modern-visible">
-                    <div class="u-h meta__number" data-discussion-id="@crosswordPage.content.content.discussionId" data-commentcount-format="content" data-discussion-closed="@{crosswordPage.content.trail.isCommentable}">
+                    <div class="meta__number">
+                        <a href="#comments" data-link-name="Comment count" class="commentcount2 tone-colour">
+                            <h3 class="commentcount2__heading">@fragments.inlineSvg("comment-16", "icon", List("inline-tone-fill")) Comments</h3>
+                            <comment-count class="commentcount2__value tone-colour" data-discussion-id="@crosswordPage.content.content.discussionId" data-discussion-closed="@{crosswordPage.content.trail.isCommentable}"></comment-count>
```

Comment:
Had a look at the [spec](https://www.w3.org/TR/custom-elements/), very informative, quote

> An autonomous custom element does not have any special meaning: it represents its children. A customized built-in element inherits the semantics of the element that it extends.

> Any namespace-less attribute that is relevant to the element's functioning, as determined by the element's author, may be specified on an autonomous custom element, so long as the attribute name is XML-compatible and contains no uppercase ASCII letters. The exception is the is attribute, which must not be specified on an autonomous custom element (and which will have no effect if it is).

> Customized built-in elements follow the normal requirements for attributes, based on the elements they extend. To add custom attribute-based behavior, use data-* attributes.



In short

```
<custom-element whatever="data" />
<button is="custom" data-whatever="data" />
```